test/emqx_auth_http_SUITE.erl

```diff
@@ -67,31 +72,30 @@ set_special_configs(_App) ->
 t_check_acl(_) ->
     %ct:pal("all configs: ~p ", [application:get_all_env(?APP)]),
     %ct:pal("emqx all configs: ~p ", [application:get_all_env(emqx)]),
-    SuperUser = #{client_id => <<"superclient">>, username => <<"superuser">>,
-                  peername => {{127, 0, 0, 1}, 2982}, zone => external},
+    SuperUser = ?USER(<<"superclient">>, <<"superuser">>, {{127,0,0,1}, 1883}, {{127, 0, 0, 1}, 2982}, external),
     deny = emqx_access_control:check_acl(SuperUser, subscribe, <<"users/testuser/1">>),
     deny = emqx_access_control:check_acl(SuperUser, publish, <<"anytopic">>),
 
-    User1 = #{client_id => <<"client1">>, username => <<"testuser">>, peername => {{127,0,0,1}, 2981}, zone => external},
-    UnIpUser1 = #{client_id => <<"client1">>, username => <<"testuser">>, peername => {{192,168,0,4}, 2981}, zone => external},
-    UnClientIdUser1 = #{client_id => <<"unkonwc">>, username => <<"testuser">>, peername => {{127,0,0,1}, 2981}, zone => external},
-    UnnameUser1= #{client_id => <<"client1">>, username => <<"unuser">>, peername => {{127,0,0,1}, 2981}, zone => external},
+    User1 = ?USER(<<"client1">>, <<"testuser">>, {{127,0,0,1}, 1883}, {{127,0,0,1}, 2981}, external),
+    UnIpUser1 = ?USER(<<"client1">>, <<"testuser">>, {{127,0,0,1}, 1883}, {{192,168,0,4}, 2981}, external),
+    UnClientIdUser1 = ?USER(<<"unkonwc">>, <<"testuser">>, {{127,0,0,1}, 1883}, {{127,0,0,1}, 2981}, external),
+    UnnameUser1= ?USER(<<"client1">>, <<"unuser">>, {{127,0,0,1}, 1883}, {{127,0,0,1}, 2981}, external),
```

Comment:
I suppose it is more clear if you do not use `macro` like `?USER` and directly write map structure here. Anyway, it's a matter of preference. Don't care about it.